.github/workflows/pr-check.yml

```diff
@@ -112,6 +112,20 @@ jobs:
           cache: 'npm'
           cache-dependency-path: ${{ env.special-working-directory-relative }}/package-lock.json
 
+      - name: Use Python 3.8
+        uses: actions/setup-python@v5
+        with:
+          python-version: '3.8'
```

Comment:
would we be fine with just testing with 3.8? or should we add more

Reply:
This is just for installing packages, we test all python from 3.8-3.12. We need to use min supported python to install packages.
![image](https://github.com/microsoft/vscode-black-formatter/assets/3840081/6fa2e543-a85f-4df2-9494-a7e1a2a2d968)

For typescript tests we don't need to test against other versions of python.

Reply:
make sense, thank you!